store/index.ts

```diff
@@ -71,27 +87,32 @@ export const mutations: MutationTree<RootState> = {
   },
   SOCKET_RECONNECT_ERROR(state: RootState) {
     Vue.set(state.socket, 'reconnectError', true);
+  },
+  SET_SYNC_STATUS(state: RootState, value: boolean) {
+    state.syncing = value;
   }
 };
 
 export const actions: ActionTree<RootState, RootState> = {
   async reset({ dispatch }, { clearCritical }: { clearCritical: boolean }) {
     const promises = [];
     promises.push(dispatch('backdrop/clearAllBackdrop', { root: true }));
-    promises.push(dispatch('deviceProfile/clearDeviceProfile', { root: true }));
-    promises.push(dispatch('displayPreferences/resetState', { root: true }));
-    promises.push(dispatch('items/clearState', { root: true }));
```

Comment:
I think this is missing, isn't it?

Reply:
Talking about the item clear

Reply:
Seems like it was eaten during a rebase :+1: